# Detailed Changelog for EDAM
Data were generated using the [Bubastis](http://www.ebi.ac.uk/efo/bubastis/) ontology-diff tool.  

# EDAM\_1.12.owl
Class: http://edamontology.org/data_0872
Label: Phylogenetic tree
-   'Phylogenetic tree' SubClassOf 'Data'
+   'Phylogenetic tree' SubClassOf 'Phylogenetic data'
 
Class: http://edamontology.org/data_1597
Label: Codon usage table
-   'Codon usage table' SubClassOf 'Data'
+   'Codon usage table' SubClassOf 'Codon usage data'
 
Class: http://edamontology.org/operation_2990
Label: Classification
-   'Classification' SubClassOf 'Analysis'
+   'Classification' SubClassOf 'Operation'
 
Class: http://edamontology.org/data_2337
Label: Resource metadata
-   'Resource metadata' SubClassOf 'Data'
+   'Resource metadata' SubClassOf 'Report'
 
Class: http://edamontology.org/operation_2928
Label: Alignment
-   'Alignment' SubClassOf 'Analysis'
+   'Alignment' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2932
Label: Hopp and Woods plotting
-   'Hopp and Woods plotting' SubClassOf 'Peptide immunogenicity prediction'
-   'Hopp and Woods plotting' SubClassOf 'Plotting'
+   'Hopp and Woods plotting' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/format_3604
Label: svg
-   'svg' SubClassOf 'Binary format'
+   'svg' SubClassOf 'XML'
 
Class: http://edamontology.org/operation_3501
Label: Enrichment
-   'Enrichment' SubClassOf 'Analysis'
+   'Enrichment' SubClassOf 'Operation'
 
Class: http://edamontology.org/data_1549
Label: Protein hydrogen bonds
-   'Protein hydrogen bonds' SubClassOf 'Protein residue interactions'
+   'Protein hydrogen bonds' SubClassOf 'Protein interaction report'
 
Class: http://edamontology.org/data_1546
Label: Protein distance matrix
-   'Protein distance matrix' SubClassOf 'Protein residue interactions'
+   'Protein distance matrix' SubClassOf 'Protein interaction report'
+   'Protein distance matrix' SubClassOf 'Distance matrix'
 
Class: http://edamontology.org/data_1548
Label: Protein residue 3D cluster
-   'Protein residue 3D cluster' SubClassOf 'Protein residue interactions'
+   'Protein residue 3D cluster' SubClassOf 'Protein interaction report'
 
Class: http://edamontology.org/data_1547
Label: Protein contact map
-   'Protein contact map' SubClassOf 'Protein residue interactions'
+   'Protein contact map' SubClassOf 'Protein interaction report'
 
Class: http://edamontology.org/data_1542
Label: Protein solvent accessibility
-   'Protein solvent accessibility' SubClassOf 'Protein residue interactions'
+   'Protein solvent accessibility' SubClassOf 'Protein structure report'
 
Class: http://edamontology.org/data_1540
Label: Protein residue interactions
-   'Protein residue interactions' SubClassOf 'Protein property'
-   'Protein residue interactions' SubClassOf 'has topic' some 'Protein folding, stability and design'
+   'Protein residue interactions' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3545
Label: Mathematical modelling
-   'Mathematical modelling' SubClassOf 'Modelling and simulation'
+   'Mathematical modelling' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_1491
Label: Structure alignment (nucleic acid pair)
-   'Structure alignment (nucleic acid pair)' SubClassOf 'Structure alignment (pair)'
-   'Structure alignment (nucleic acid pair)' SubClassOf 'Structure alignment (nucleic acid)'
+   'Structure alignment (nucleic acid pair)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_1483
Label: Structure alignment (protein pair)
-   'Structure alignment (protein pair)' SubClassOf 'Structure alignment (protein)'
-   'Structure alignment (protein pair)' SubClassOf 'Structure alignment (pair)'
+   'Structure alignment (protein pair)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_2402
Label: Protein-drug interaction report
-   'Protein-drug interaction report' SubClassOf 'Protein structure report'
-   'Protein-drug interaction report' SubClassOf 'Protein-ligand interaction report'
-   'Protein-drug interaction report' SubClassOf 'Drug report'
+   'Protein-drug interaction report' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3435
Label: Standardization and normalization
-   'Standardization and normalization' SubClassOf 'Analysis'
+   'Standardization and normalization' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3434
Label: Conversion
-   'Conversion' SubClassOf 'Utility operation'
+   'Conversion' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3439
Label: Pathway or network prediction
-   'Pathway or network prediction' SubClassOf 'Pathway or network processing'
+   'Pathway or network prediction' SubClassOf 'Pathway or network analysis'
 
Class: http://edamontology.org/operation_3438
Label: Calculation
-   'Calculation' SubClassOf 'Analysis'
+   'Calculation' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3433
Label: Assembly
-   'Assembly' SubClassOf 'Analysis'
+   'Assembly' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3432
Label: Clustering
-   'Clustering' SubClassOf 'Analysis'
+   'Clustering' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3440
Label: Genome assembly
-   'Genome assembly' SubClassOf 'Sequence assembly'
+   'Genome assembly' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3441
Label: Plotting
-   'Plotting' SubClassOf 'Analysis'
+   'Plotting' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3443
Label: Image analysis
-   'Image analysis' SubClassOf 'Operation (typed)'
+   'Image analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_3465
Label: Correlation
-   'Correlation' SubClassOf 'Analysis'
+   'Correlation' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_3429
Label: Generation
-   'Generation' SubClassOf 'Analysis'
+   'Generation' SubClassOf 'Operation'
 
Class: http://edamontology.org/data_3546
Label: Image metadata
-   'Image metadata' SubClassOf 'Data'
+   'Image metadata' SubClassOf 'Report'
 
Class: http://edamontology.org/format_1964
Label: plain text format (unformatted)
-   'plain text format (unformatted)' SubClassOf 'Sequence record format (text)'
 
Class: http://edamontology.org/data_1386
Label: Sequence alignment (nucleic acid pair)
-   'Sequence alignment (nucleic acid pair)' SubClassOf 'Sequence alignment (pair)'
-   'Sequence alignment (nucleic acid pair)' SubClassOf 'Sequence alignment (nucleic acid)'
+   'Sequence alignment (nucleic acid pair)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_1387
Label: Sequence alignment (protein pair)
-   'Sequence alignment (protein pair)' SubClassOf 'Sequence alignment (pair)'
-   'Sequence alignment (protein pair)' SubClassOf 'Sequence alignment (protein)'
+   'Sequence alignment (protein pair)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_0180
Label: Protein fold recognition
-   'Protein fold recognition' SubClassOf 'Structure prediction'
+   'Protein fold recognition' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_0199
Label: Genetic variation
-   'Genetic variation' SubClassOf 'Genotype and phenotype'
+   'Genetic variation' SubClassOf 'Genetics'
 
Class: http://edamontology.org/topic_0160
Label: Sequence sites, features and motifs
-   'Sequence sites, features and motifs' SubClassOf 'Sequence analysis'
+   'Sequence sites, features and motifs' SubClassOf 'Computational biology'
 
Class: http://edamontology.org/topic_0172
Label: Protein structure prediction
-   'Protein structure prediction' SubClassOf 'Protein structure analysis'
-   'Protein structure prediction' SubClassOf 'Structure prediction'
+   'Protein structure prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_0173
Label: Nucleic acid structure prediction
-   'Nucleic acid structure prediction' SubClassOf 'Structure prediction'
-   'Nucleic acid structure prediction' SubClassOf 'Nucleic acid structure analysis'
+   'Nucleic acid structure prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_0177
Label: Molecular docking
-   'Molecular docking' SubClassOf 'Structure prediction'
+   'Molecular docking' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0559
Label: Immunogenicity prediction
-   'Immunogenicity prediction' SubClassOf 'has output' some 'Protein structure'
-   'Immunogenicity prediction' SubClassOf 'Peptide immunogenicity prediction'
-   'Immunogenicity prediction' SubClassOf 'has topic' some 'Protein folding, stability and design'
-   'Immunogenicity prediction' SubClassOf 'has topic' some 'Immunology'
+   'Immunogenicity prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3283
Label: Anonymisation
-   'Anonymisation' SubClassOf 'Utility operation'
+   'Anonymisation' SubClassOf 'Operation'
 
Class: http://edamontology.org/topic_0159
Label: Sequence comparison
-   'Sequence comparison' SubClassOf 'Sequence analysis'
+   'Sequence comparison' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3351
Label: Protein surface analysis
-   'Protein surface analysis' SubClassOf 'Protein property calculation (from structure)'
+   'Protein surface analysis' SubClassOf 'has topic' some 'Protein structural motifs and surfaces'
+   'Protein surface analysis' SubClassOf 'Structure analysis'
+   'Protein surface analysis' SubClassOf 'has topic' some 'Protein properties'
 
Class: http://edamontology.org/operation_0567
Label: Phylogenetic tree visualisation
-   'Phylogenetic tree visualisation' SubClassOf 'Phylogenetic tree processing'
+   'Phylogenetic tree visualisation' SubClassOf 'Phylogenetic tree analysis'
 
Class: http://edamontology.org/operation_0566
Label: Sequence cluster visualisation
-   'Sequence cluster visualisation' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/operation_0563
Label: Codon usage table formatting
-   'Codon usage table formatting' SubClassOf 'Formatting'
-   'Codon usage table formatting' SubClassOf 'Codon usage table processing'
-   'Codon usage table formatting' SubClassOf 'has output' some 'Codon usage table'
-   'Codon usage table formatting' SubClassOf 'has input' some 'Codon usage table'
+   'Codon usage table formatting' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0562
Label: Sequence alignment formatting
-   'Sequence alignment formatting' SubClassOf 'has input' some 'Sequence alignment'
-   'Sequence alignment formatting' SubClassOf 'has output' some 'Sequence alignment'
-   'Sequence alignment formatting' SubClassOf 'Formatting'
+   'Sequence alignment formatting' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0561
Label: Sequence formatting
-   'Sequence formatting' SubClassOf 'has input' some 'Sequence'
-   'Sequence formatting' SubClassOf 'Formatting'
-   'Sequence formatting' SubClassOf 'has output' some 'Sequence'
+   'Sequence formatting' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0560
Label: DNA vaccine design
-   'DNA vaccine design' SubClassOf 'Design'
-   'DNA vaccine design' SubClassOf 'has topic' some 'Nucleic acid structure prediction'
 
Class: http://edamontology.org/topic_3517
Label: GWAS study
-   'GWAS study' SubClassOf 'Laboratory techniques'
+   'GWAS study' SubClassOf http://edamontology.org/topic_3678
 
Class: http://edamontology.org/topic_3523
Label: RNAi experiment
-   'RNAi experiment' SubClassOf 'Sequencing'
+   'RNAi experiment' SubClassOf 'Laboratory techniques'
 
Class: http://edamontology.org/operation_2233
Label: Representative sequence identification
-   'Representative sequence identification' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/data_2991
Label: Protein torsion angle data
-   'Protein torsion angle data' SubClassOf 'Protein property'
+   'Protein torsion angle data' SubClassOf 'Protein structure report'
 
Class: http://edamontology.org/topic_1312
Label: Promoters
-   'Promoters' SubClassOf 'Gene transcription features'
+   'Promoters' SubClassOf 'Transcription factors and regulatory sites'
 
Class: http://edamontology.org/data_1235
Label: Sequence cluster
-   'Sequence cluster' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/data_1274
Label: Map
-   'Map' SubClassOf 'Data'
+   'Map' SubClassOf 'Map data'
 
Class: http://edamontology.org/operation_3191
Label: Trim to reference
-   'Trim to reference' SubClassOf 'Sequence trimming'
+   'Trim to reference' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3190
Label: Trim vector
-   'Trim vector' SubClassOf 'Sequence trimming'
+   'Trim vector' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_0971
Label: Article
-   'Article' SubClassOf 'Data'
+   'Article' SubClassOf 'Article data'
 
Class: http://edamontology.org/data_0906
Label: Protein interaction report
-   'Protein interaction report' SubClassOf 'Protein report'
+   'Protein interaction report' SubClassOf 'Protein structure report'
 
Class: http://edamontology.org/operation_3189
Label: Trim ends
-   'Trim ends' SubClassOf 'Sequence trimming'
+   'Trim ends' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3219
Label: Read pre-processing
-   'Read pre-processing' SubClassOf 'Validation'
+   'Read pre-processing' SubClassOf 'Sequencing quality control'
 
Class: http://edamontology.org/operation_3214
Label: Spectral analysis
+   'Spectral analysis' SubClassOf 'has topic' some 'Proteomics'
 
Class: http://edamontology.org/operation_3213
Label: Genome indexing (suffix arrays)
-   'Genome indexing (suffix arrays)' SubClassOf 'Genome indexing'
+   'Genome indexing (suffix arrays)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3212
Label: Genome indexing (Burrows-Wheeler)
-   'Genome indexing (Burrows-Wheeler)' SubClassOf 'Genome indexing'
+   'Genome indexing (Burrows-Wheeler)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_3229
Label: Exome analysis
-   'Exome analysis' SubClassOf 'Nucleic acid sequence analysis'
+   'Exome analysis' SubClassOf 'Sequence assembly'
 
Class: http://edamontology.org/operation_0303
Label: Protein fold recognition
-   'Protein fold recognition' SubClassOf 'has topic' some 'Protein fold recognition'
 
Class: http://edamontology.org/data_2767
Label: Identifier with metadata
-   'Identifier with metadata' SubClassOf 'Identifier'
+   'Identifier with metadata' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_2048
Label: Report
-   'Report' SubClassOf 'Resource metadata'
+   'Report' SubClassOf 'Data'
 
Class: http://edamontology.org/topic_3298
Label: Phenomics
+   'Phenomics' SubClassOf 'Omics'
 
Class: http://edamontology.org/data_1150
Label: Disease ID
-   'Disease ID' SubClassOf 'is identifier of' some 'Disease report'
-   'Disease ID' SubClassOf 'Identifier (typed)'
+   'Disease ID' SubClassOf http://edamontology.org/data_3667
 
Class: http://edamontology.org/operation_1832
Label: Residue contact calculation (residue-nucleic acid)
-   'Residue contact calculation (residue-nucleic acid)' SubClassOf 'Protein-nucleic acid binding site analysis'
-   'Residue contact calculation (residue-nucleic acid)' SubClassOf 'Residue contact calculation'
-   'Residue contact calculation (residue-nucleic acid)' SubClassOf 'Protein structural motif recognition'
+   'Residue contact calculation (residue-nucleic acid)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1834
Label: Residue contact calculation (residue-metal)
-   'Residue contact calculation (residue-metal)' SubClassOf 'Residue contact calculation'
-   'Residue contact calculation (residue-metal)' SubClassOf 'Protein binding site prediction (from structure)'
+   'Residue contact calculation (residue-metal)' SubClassOf 'Residue interaction calculation'
 
Class: http://edamontology.org/operation_1835
Label: Residue contact calculation (residue-negative ion)
-   'Residue contact calculation (residue-negative ion)' SubClassOf 'Residue contact calculation'
+   'Residue contact calculation (residue-negative ion)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1831
Label: Metal-bound cysteine detection
+   'Metal-bound cysteine detection' SubClassOf 'Residue interaction calculation'
 
Class: http://edamontology.org/operation_1826
Label: Full torsion angle calculation
-   'Full torsion angle calculation' SubClassOf 'Torsion angle calculation'
+   'Full torsion angle calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1825
Label: Backbone torsion angle calculation
-   'Backbone torsion angle calculation' SubClassOf 'Torsion angle calculation'
+   'Backbone torsion angle calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1828
Label: Tau angle calculation
-   'Tau angle calculation' SubClassOf 'Torsion angle calculation'
+   'Tau angle calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1827
Label: Cysteine torsion angle calculation
-   'Cysteine torsion angle calculation' SubClassOf 'Torsion angle calculation'
+   'Cysteine torsion angle calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1846
Label: HET group detection
-   'HET group detection' SubClassOf 'Residue contact calculation (residue-ligand)'
+   'HET group detection' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1843
Label: Residue packing validation
-   'Residue packing validation' SubClassOf 'Residue non-canonical interaction detection'
+   'Residue packing validation' SubClassOf 'Protein model validation'
 
Class: http://edamontology.org/operation_1841
Label: Rotamer likelihood prediction
-   'Rotamer likelihood prediction' SubClassOf 'Protein modelling (side chains)'
+   'Rotamer likelihood prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1842
Label: Proline mutation value calculation
-   'Proline mutation value calculation' SubClassOf 'Protein modelling (mutation)'
+   'Proline mutation value calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1839
Label: Salt bridge calculation
-   'Salt bridge calculation' SubClassOf 'Residue contact calculation (residue-residue)'
+   'Salt bridge calculation' SubClassOf 'Residue interaction calculation'
 
Class: http://edamontology.org/operation_1838
Label: Residue contact calculation (residue-ligand)
-   'Residue contact calculation (residue-ligand)' SubClassOf 'Protein binding site prediction (from structure)'
-   'Residue contact calculation (residue-ligand)' SubClassOf 'Residue contact calculation'
+   'Residue contact calculation (residue-ligand)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1837
Label: Residue symmetry contact calculation
-   'Residue symmetry contact calculation' SubClassOf 'Residue contact calculation (residue-residue)'
+   'Residue symmetry contact calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1836
Label: Residue bump detection
-   'Residue bump detection' SubClassOf 'Residue non-canonical interaction detection'
+   'Residue bump detection' SubClassOf 'Protein model validation'
 
Class: http://edamontology.org/operation_1820
Label: Protein residue surface calculation (vacuum accessible)
-   'Protein residue surface calculation (vacuum accessible)' SubClassOf 'Protein residue surface calculation'
+   'Protein residue surface calculation (vacuum accessible)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1823
Label: Protein surface calculation (accessible molecular)
-   'Protein surface calculation (accessible molecular)' SubClassOf 'Protein surface calculation'
+   'Protein surface calculation (accessible molecular)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1824
Label: Protein surface calculation (accessible)
-   'Protein surface calculation (accessible)' SubClassOf 'Protein surface calculation'
+   'Protein surface calculation (accessible)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1821
Label: Protein residue surface calculation (accessible molecular)
-   'Protein residue surface calculation (accessible molecular)' SubClassOf 'Protein residue surface calculation'
+   'Protein residue surface calculation (accessible molecular)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1822
Label: Protein residue surface calculation (vacuum molecular)
-   'Protein residue surface calculation (vacuum molecular)' SubClassOf 'Protein residue surface calculation'
+   'Protein residue surface calculation (vacuum molecular)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1817
Label: Protein atom surface calculation (accessible)
-   'Protein atom surface calculation (accessible)' SubClassOf 'Protein atom surface calculation'
+   'Protein atom surface calculation (accessible)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1816
Label: Surface rendering
-   'Surface rendering' SubClassOf 'Protein surface calculation'
+   'Surface rendering' SubClassOf 'Protein surface analysis'
 
Class: http://edamontology.org/operation_1819
Label: Protein residue surface calculation (accessible)
-   'Protein residue surface calculation (accessible)' SubClassOf 'Protein residue surface calculation'
+   'Protein residue surface calculation (accessible)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_1818
Label: Protein atom surface calculation (accessible molecular)
-   'Protein atom surface calculation (accessible molecular)' SubClassOf 'Protein atom surface calculation'
+   'Protein atom surface calculation (accessible molecular)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2495
Label: Gene expression data analysis
-   'Gene expression data analysis' SubClassOf 'Operation (typed)'
+   'Gene expression data analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_2497
Label: Pathway or network analysis
-   'Pathway or network analysis' SubClassOf 'has input' some 'Pathway or network'
-   'Pathway or network analysis' SubClassOf 'Pathway or network processing'
+   'Pathway or network analysis' SubClassOf 'Analysis'
+   'Pathway or network analysis' SubClassOf 'has topic' some 'Molecular interactions, pathways and networks'
 
Class: http://edamontology.org/operation_2491
Label: Hydrogen bond calculation (inter-residue)
-   'Hydrogen bond calculation (inter-residue)' SubClassOf 'Hydrogen bond calculation'
-   'Hydrogen bond calculation (inter-residue)' SubClassOf 'Residue contact calculation (residue-residue)'
+   'Hydrogen bond calculation (inter-residue)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2490
Label: Residue contact calculation (residue-residue)
-   'Residue contact calculation (residue-residue)' SubClassOf 'Residue contact calculation'
+   'Residue contact calculation (residue-residue)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2480
Label: Structure analysis
-   'Structure analysis' SubClassOf 'Operation (typed)'
+   'Structure analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_3083
Label: Pathway or network visualisation
-   'Pathway or network visualisation' SubClassOf 'Pathway or network processing'
+   'Pathway or network visualisation' SubClassOf 'Pathway or network analysis'
 
Class: http://edamontology.org/operation_3096
Label: Editing
-   'Editing' SubClassOf 'Utility operation'
+   'Editing' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_1848
Label: Structure formatting
-   'Structure formatting' SubClassOf 'Formatting'
+   'Structure formatting' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2460
Label: Protein atom surface calculation
-   'Protein atom surface calculation' SubClassOf 'Protein surface and interior calculation'
+   'Protein atom surface calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2461
Label: Protein residue surface calculation
-   'Protein residue surface calculation' SubClassOf 'Protein surface and interior calculation'
+   'Protein residue surface calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2464
Label: Protein-protein interaction prediction
+   'Protein-protein interaction prediction' SubClassOf 'Protein function prediction'
 
Class: http://edamontology.org/operation_2462
Label: Protein surface calculation
-   'Protein surface calculation' SubClassOf 'Protein surface and interior calculation'
+   'Protein surface calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2451
Label: Sequence comparison
-   'Sequence comparison' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/operation_0385
Label: Protein hydropathy cluster calculation
-   'Protein hydropathy cluster calculation' SubClassOf 'Protein hydropathy calculation (from structure)'
-   'Protein hydropathy cluster calculation' SubClassOf 'Protein residue cluster calculation'
+   'Protein hydropathy cluster calculation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0384
Label: Protein solvent accessibility calculation
-   'Protein solvent accessibility calculation' SubClassOf 'Protein property calculation (from structure)'
-   'Protein solvent accessibility calculation' SubClassOf 'has topic' some 'Protein properties'
+   'Protein solvent accessibility calculation' SubClassOf 'Protein surface analysis'
 
Class: http://edamontology.org/operation_0383
Label: Protein hydropathy calculation (from structure)
-   'Protein hydropathy calculation (from structure)' SubClassOf 'Protein hydropathy calculation'
-   'Protein hydropathy calculation (from structure)' SubClassOf 'Protein property calculation (from structure)'
-   'Protein hydropathy calculation (from structure)' SubClassOf 'Protein structure analysis'
+   'Protein hydropathy calculation (from structure)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0388
Label: Protein binding site prediction (from structure)
-   'Protein binding site prediction (from structure)' SubClassOf 'Protein structural motif recognition'
-   'Protein binding site prediction (from structure)' SubClassOf 'Protein binding site prediction'
+   'Protein binding site prediction (from structure)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0387
Label: Protein surface and interior calculation
-   'Protein surface and interior calculation' SubClassOf 'has topic' some 'Protein structural motifs and surfaces'
-   'Protein surface and interior calculation' SubClassOf 'Protein solvent accessibility calculation'
+   'Protein surface and interior calculation' SubClassOf 'Protein surface analysis'
 
Class: http://edamontology.org/operation_0391
Label: Protein distance matrix calculation
-   'Protein distance matrix calculation' SubClassOf 'Residue interaction calculation'
+   'Protein distance matrix calculation' SubClassOf 'Residue contact calculation'
 
Class: http://edamontology.org/operation_2443
Label: Phylogenetic tree processing
-   'Phylogenetic tree processing' SubClassOf 'has topic' some 'Phylogeny'
-   'Phylogenetic tree processing' SubClassOf 'Operation (typed)'
+   'Phylogenetic tree processing' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0393
Label: Protein residue cluster calculation
-   'Protein residue cluster calculation' SubClassOf 'Residue contact calculation (residue-residue)'
+   'Protein residue cluster calculation' SubClassOf 'Residue contact calculation'
 
Class: http://edamontology.org/operation_0392
Label: Protein contact map calculation
-   'Protein contact map calculation' SubClassOf 'Residue contact calculation (residue-residue)'
+   'Protein contact map calculation' SubClassOf 'Protein distance matrix calculation'
 
Class: http://edamontology.org/operation_0395
Label: Residue non-canonical interaction detection
-   'Residue non-canonical interaction detection' SubClassOf 'Protein model validation'
-   'Residue non-canonical interaction detection' SubClassOf 'Residue interaction calculation'
+   'Residue non-canonical interaction detection' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2430
Label: Design
-   'Design' SubClassOf 'Analysis'
+   'Design' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2438
Label: Pathway or network processing
-   'Pathway or network processing' SubClassOf 'has topic' some 'Molecular interactions, pathways and networks'
-   'Pathway or network processing' SubClassOf 'Operation (typed)'
+   'Pathway or network processing' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2420
Label: Operation (typed)
-   'Operation (typed)' SubClassOf 'Operation'
+   'Operation (typed)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_2423
Label: Prediction and recognition
-   'Prediction and recognition' SubClassOf 'Analysis'
+   'Prediction and recognition' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2425
Label: Optimisation and refinement
-   'Optimisation and refinement' SubClassOf 'Analysis'
+   'Optimisation and refinement' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2424
Label: Comparison
-   'Comparison' SubClassOf 'Analysis'
+   'Comparison' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2426
Label: Modelling and simulation
-   'Modelling and simulation' SubClassOf 'Analysis'
+   'Modelling and simulation' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2429
Label: Mapping
-   'Mapping' SubClassOf 'Analysis'
+   'Mapping' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2428
Label: Validation
-   'Validation' SubClassOf 'Analysis'
+   'Validation' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_2409
Label: Utility operation
-   'Utility operation' SubClassOf 'Operation'
+   'Utility operation' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_2403
Label: Sequence analysis
-   'Sequence analysis' SubClassOf 'Operation (typed)'
+   'Sequence analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_0325
Label: Phylogenetic tree comparison
-   'Phylogenetic tree comparison' SubClassOf 'Phylogenetic tree processing'
+   'Phylogenetic tree comparison' SubClassOf 'Phylogenetic tree analysis'
 
Class: http://edamontology.org/operation_0326
Label: Phylogenetic tree editing
-   'Phylogenetic tree editing' SubClassOf 'Phylogenetic tree processing'
+   'Phylogenetic tree editing' SubClassOf 'Phylogenetic tree analysis'
 
Class: http://edamontology.org/operation_0323
Label: Phylogenetic tree generation
-   'Phylogenetic tree generation' SubClassOf 'Phylogenetic tree processing'
+   'Phylogenetic tree generation' SubClassOf 'Phylogenetic tree analysis'
 
Class: http://edamontology.org/operation_0324
Label: Phylogenetic tree analysis
-   'Phylogenetic tree analysis' SubClassOf 'has output' some 'Phylogenetic data'
-   'Phylogenetic tree analysis' SubClassOf 'Phylogenetic tree processing'
-   'Phylogenetic tree analysis' SubClassOf 'has input' some 'Phylogenetic tree'
+   'Phylogenetic tree analysis' SubClassOf 'has topic' some 'Phylogeny'
+   'Phylogenetic tree analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_0321
Label: Protein model validation
-   'Protein model validation' SubClassOf 'has topic' some 'Protein structure prediction'
 
Class: http://edamontology.org/operation_0336
Label: Format validation
-   'Format validation' SubClassOf 'Utility operation'
 
Class: http://edamontology.org/operation_0338
Label: Sequence database search
-   'Sequence database search' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/operation_0330
Label: Protein SNP mapping
-   'Protein SNP mapping' SubClassOf 'Mapping'
-   'Protein SNP mapping' SubClassOf 'Protein modelling (mutation)'
-   'Protein SNP mapping' SubClassOf 'has topic' some 'SNP'
+   'Protein SNP mapping' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0410
Label: Protein crystallizability prediction
-   'Protein crystallizability prediction' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein crystallizability prediction' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0413
Label: MHC peptide immunogenicity prediction
-   'MHC peptide immunogenicity prediction' SubClassOf 'Peptide immunogenicity prediction'
+   'MHC peptide immunogenicity prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0408
Label: Protein globularity prediction
-   'Protein globularity prediction' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein globularity prediction' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0409
Label: Protein solubility prediction
-   'Protein solubility prediction' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein solubility prediction' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0406
Label: Protein aliphatic index calculation
-   'Protein aliphatic index calculation' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein aliphatic index calculation' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0407
Label: Protein hydrophobic moment plotting
-   'Protein hydrophobic moment plotting' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein hydrophobic moment plotting' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0401
Label: Protein hydropathy calculation (from sequence)
-   'Protein hydropathy calculation (from sequence)' SubClassOf 'Protein hydropathy calculation'
-   'Protein hydropathy calculation (from sequence)' SubClassOf 'Protein sequence analysis'
+   'Protein hydropathy calculation (from sequence)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0405
Label: Protein hydrophobic region calculation
-   'Protein hydrophobic region calculation' SubClassOf 'Protein hydropathy calculation (from sequence)'
+   'Protein hydrophobic region calculation' SubClassOf 'Protein hydropathy calculation'
 
Class: http://edamontology.org/operation_0438
Label: Regulatory element prediction
-   'Regulatory element prediction' SubClassOf 'Gene component prediction'
+   'Regulatory element prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_0437
Label: Selenocysteine insertion sequence (SECIS) prediction
-   'Selenocysteine insertion sequence (SECIS) prediction' SubClassOf 'Gene component prediction'
+   'Selenocysteine insertion sequence (SECIS) prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_0436
Label: Coding region prediction
-   'Coding region prediction' SubClassOf 'Gene component prediction'
+   'Coding region prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_0435
Label: Operon prediction
-   'Operon prediction' SubClassOf 'Whole gene prediction'
+   'Operon prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_0434
Label: Integrated gene prediction
-   'Integrated gene prediction' SubClassOf 'Whole gene prediction'
+   'Integrated gene prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_0429
Label: Quadruplex formation site detection
-   'Quadruplex formation site detection' SubClassOf 'has topic' some 'Nucleic acid structure prediction'
 
Class: http://edamontology.org/operation_0425
Label: Whole gene prediction
-   'Whole gene prediction' SubClassOf 'Gene prediction'
+   'Whole gene prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0426
Label: Gene component prediction
-   'Gene component prediction' SubClassOf 'Gene prediction'
+   'Gene component prediction' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0420
Label: Protein-nucleic acid binding prediction
-   'Protein-nucleic acid binding prediction' SubClassOf 'Protein binding site prediction (from sequence)'
+   'Protein-nucleic acid binding prediction' SubClassOf 'Protein binding site prediction'
 
Class: http://edamontology.org/topic_3346
Label: Sequence search
-   'Sequence search' SubClassOf 'Sequence analysis'
+   'Sequence search' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_3344
Label: Biomedical science
-   'Biomedical science' SubClassOf 'Biology'
-   'Biomedical science' SubClassOf 'Medicine'
+   'Biomedical science' SubClassOf 'Topic'
 
Class: http://edamontology.org/operation_0419
Label: Protein binding site prediction (from sequence)
-   'Protein binding site prediction (from sequence)' SubClassOf 'Protein sequence feature detection'
-   'Protein binding site prediction (from sequence)' SubClassOf 'Protein binding site prediction'
+   'Protein binding site prediction (from sequence)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_3307
Label: Computational biology
-   'Computational biology' SubClassOf 'Biology'
-   'Computational biology' SubClassOf 'Computer science'
+   'Computational biology' SubClassOf 'Topic'
 
Class: http://edamontology.org/topic_3391
Label: Omics
-   'Omics' SubClassOf 'Biology'
+   'Omics' SubClassOf 'Topic'
 
Class: http://edamontology.org/topic_3379
Label: Preclinical and clinical studies
+   'Preclinical and clinical studies' SubClassOf http://edamontology.org/topic_3678
 
Class: http://edamontology.org/operation_0483
Label: Structured RNA prediction and optimisation
-   'Structured RNA prediction and optimisation' SubClassOf 'has topic' some 'Nucleic acid structure prediction'
 
Class: http://edamontology.org/topic_0594
Label: Sequence classification
-   'Sequence classification' SubClassOf 'Sequence analysis'
+   'Sequence classification' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0464
Label: tRNA gene prediction
-   'tRNA gene prediction' SubClassOf 'Whole gene prediction'
+   'tRNA gene prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_1913
Label: Residue validation
-   'Residue validation' SubClassOf 'Protein model validation'
+   'Residue validation' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0474
Label: Protein structure prediction
-   'Protein structure prediction' SubClassOf 'has topic' some 'Protein structure prediction'
 
Class: http://edamontology.org/operation_0475
Label: Nucleic acid structure prediction
-   'Nucleic acid structure prediction' SubClassOf 'has topic' some 'Nucleic acid structure analysis'
 
Class: http://edamontology.org/operation_0478
Label: Molecular docking
-   'Molecular docking' SubClassOf 'has topic' some 'Molecular docking'
 
Class: http://edamontology.org/topic_3123
Label: Expression signals
-   'Expression signals' SubClassOf 'Gene structure'
-   'Expression signals' SubClassOf 'Gene expression'
+   'Expression signals' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/data_1646
Label: Molecular weights standard fingerprint
-   'Molecular weights standard fingerprint' SubClassOf 'Peptide mass fingerprint'
+   'Molecular weights standard fingerprint' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/topic_3169
Label: ChIP-seq
-   'ChIP-seq' SubClassOf 'Obsolete concept (EDAM)'
+   'ChIP-seq' SubClassOf 'Sequencing'
+   'ChIP-seq' SubClassOf http://edamontology.org/topic_3656
 
Class: http://edamontology.org/data_0582
Label: Ontology
-   'Ontology' SubClassOf 'Data'
+   'Ontology' SubClassOf 'Ontology data'
 
Class: http://edamontology.org/topic_3179
Label: ChIP-on-chip
+   'ChIP-on-chip' SubClassOf http://edamontology.org/topic_3656
 
Class: http://edamontology.org/topic_3170
Label: RNA-Seq
-   'RNA-Seq' SubClassOf 'Obsolete concept (EDAM)'
+   'RNA-Seq' SubClassOf 'Sequencing'
 
Class: http://edamontology.org/topic_0749
Label: Transcription factors and regulatory sites
-   'Transcription factors and regulatory sites' SubClassOf 'Promoters'
+   'Transcription factors and regulatory sites' SubClassOf 'Gene transcription features'
 
Class: http://edamontology.org/operation_0291
Label: Sequence clustering
-   'Sequence clustering' SubClassOf 'has topic' some 'Sequence comparison'
 
Class: http://edamontology.org/operation_0299
Label: 3D profile-to-3D profile alignment
-   '3D profile-to-3D profile alignment' SubClassOf 'Alignment'
 
Class: http://edamontology.org/topic_0769
Label: Workflows
-   'Workflows' SubClassOf 'Obsolete concept (EDAM)'
+   'Workflows' SubClassOf 'Data management'
 
Class: http://edamontology.org/operation_2501
Label: Nucleic acid analysis
-   'Nucleic acid analysis' SubClassOf 'Operation (typed)'
+   'Nucleic acid analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_2502
Label: Protein analysis
-   'Protein analysis' SubClassOf 'has topic' some 'Proteins'
-   'Protein analysis' SubClassOf 'Operation (typed)'
+   'Protein analysis' SubClassOf 'has topic' some 'Proteomics'
+   'Protein analysis' SubClassOf 'Analysis'
 
Class: http://edamontology.org/operation_0226
Label: Annotation
-   'Annotation' SubClassOf 'Analysis'
+   'Annotation' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_0227
Label: Indexing
-   'Indexing' SubClassOf 'Utility operation'
+   'Indexing' SubClassOf 'Operation'
 
Class: http://edamontology.org/operation_0248
Label: Residue interaction calculation
-   'Residue interaction calculation' SubClassOf 'has output' some 'Protein residue interactions'
+   'Residue interaction calculation' SubClassOf 'Residue interaction calculation'
 
Class: http://edamontology.org/operation_0250
Label: Protein property calculation
-   'Protein property calculation' SubClassOf 'has output' some 'Protein property'
-   'Protein property calculation' SubClassOf 'has topic' some 'Protein properties'
 
Class: http://edamontology.org/operation_0267
Label: Protein secondary structure prediction
-   'Protein secondary structure prediction' SubClassOf 'has topic' some 'Protein structure prediction'
 
Class: http://edamontology.org/operation_2575
Label: Protein binding site prediction
+   'Protein binding site prediction' SubClassOf 'Protein feature detection'
 
Class: http://edamontology.org/operation_0278
Label: RNA secondary structure prediction
-   'RNA secondary structure prediction' SubClassOf 'has topic' some 'Nucleic acid structure prediction'
 
Class: http://edamontology.org/operation_0279
Label: Nucleic acid folding prediction
-   'Nucleic acid folding prediction' SubClassOf 'has topic' some 'Nucleic acid structure prediction'
 
Class: http://edamontology.org/operation_0274
Label: Protein-protein interaction prediction (from protein sequence)
-   'Protein-protein interaction prediction (from protein sequence)' SubClassOf 'Protein function prediction'
-   'Protein-protein interaction prediction (from protein sequence)' SubClassOf 'Protein-protein interaction prediction'
+   'Protein-protein interaction prediction (from protein sequence)' SubClassOf 'Obsolete concept (EDAM)'
 
Class: http://edamontology.org/operation_0275
Label: Protein-protein interaction prediction (from protein structure)
-   'Protein-protein interaction prediction (from protein structure)' SubClassOf 'Protein-protein interaction prediction'
-   'Protein-protein interaction prediction (from protein structure)' SubClassOf 'Protein structure analysis'
-   'Protein-protein interaction prediction (from protein structure)' SubClassOf 'Protein feature detection'
+   'Protein-protein interaction prediction (from protein structure)' SubClassOf 'Obsolete concept (EDAM)'
Back to top
New classes:
 
Class: http://edamontology.org/format_3665
Label: K-mer countgraph
+   'K-mer countgraph' SubClassOf 'Graph format'
+   'K-mer countgraph' SubClassOf 'Binary format'
 
Class: http://edamontology.org/format_3652
Label: dta
+   'dta' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/format_3653
Label: pkl
+   'pkl' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/format_3650
Label: netCDF
+   'netCDF' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/format_3651
Label: MGF
+   'MGF' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/format_3657
Label: GPML
+   'GPML' SubClassOf 'Biological pathway or network format'
+   'GPML' SubClassOf 'XML'
 
Class: http://edamontology.org/format_3654
Label: mzXML
+   'mzXML' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/format_3655
Label: pepXML
+   'pepXML' SubClassOf 'Mass spectrometry data format'
 
Class: http://edamontology.org/topic_3656
Label: Immunoprecipitation experiment
+   'Immunoprecipitation experiment' SubClassOf 'Laboratory techniques'
 
Class: http://edamontology.org/topic_3676
Label: Exome sequencing
+   'Exome sequencing' SubClassOf 'Sequencing'
 
Class: http://edamontology.org/topic_3679
Label: Animal study
+   'Animal study' SubClassOf 'Experimental design and studies'
+   'Animal study' SubClassOf 'Laboratory animal science'
 
Class: http://edamontology.org/topic_3678
Label: Experimental design and studies
+   'Experimental design and studies' SubClassOf 'Topic'
 
Class: http://edamontology.org/topic_3673
Label: Whole genome sequencing
+   'Whole genome sequencing' SubClassOf 'Sequencing'
 
Class: http://edamontology.org/topic_3674
Label: Methylated DNA immunoprecipitation
+   'Methylated DNA immunoprecipitation' SubClassOf 'Immunoprecipitation experiment'
 
Class: http://edamontology.org/format_3626
Label: MAT
+   'MAT' SubClassOf 'is format of' some '3D-1D scoring matrix'
+   'MAT' SubClassOf 'Matrix format'
 
Class: http://edamontology.org/data_3671
Label: Text
+   'Text' SubClassOf 'Parameter'
+   'Text' SubClassOf 'Article data'
 
Class: http://edamontology.org/data_3670
Label: Online course
+   'Online course' SubClassOf 'Training material'
 
Class: http://edamontology.org/data_3667
Label: Disease identifier
+   'Disease identifier' SubClassOf 'is identifier of' some 'Disease report'
+   'Disease identifier' SubClassOf 'Accession'
+   'Disease identifier' SubClassOf 'Identifier (typed)'
 
Class: http://edamontology.org/data_3669
Label: Training material
+   'Training material' SubClassOf 'Data'
 
Class: http://edamontology.org/data_3668
Label: Disease name
+   'Disease name' SubClassOf 'Name'
+   'Disease name' SubClassOf 'Disease identifier'
 
Class: http://edamontology.org/topic_3557
Label: Protein interaction experiment
+   'Protein interaction experiment' SubClassOf 'Laboratory techniques'
 
Class: http://edamontology.org/operation_3642
Label: Dimethyl
+   'Dimethyl' SubClassOf 'Labeled quantification'
 
Class: http://edamontology.org/operation_3641
Label: TMT-tag
+   'TMT-tag' SubClassOf 'Labeled quantification'
 
Class: http://edamontology.org/operation_3640
Label: 18O labeling
+   '18O labeling' SubClassOf 'Labeled quantification'
 
Class: http://edamontology.org/operation_3646
Label: Peptide database search
+   'Peptide database search' SubClassOf 'Peptide identification'
+   'Peptide database search' SubClassOf 'Database search'
 
Class: http://edamontology.org/operation_3645
Label: PTM identification
+   'PTM identification' SubClassOf 'Peptide identification'
 
Class: http://edamontology.org/operation_3644
Label: de Novo sequencing
+   'de Novo sequencing' SubClassOf 'Sequence generation (protein)'
+   'de Novo sequencing' SubClassOf 'Peptide identification'
 
Class: http://edamontology.org/operation_3643
Label: Tag-based peptide identification
+   'Tag-based peptide identification' SubClassOf 'Peptide identification'
 
Class: http://edamontology.org/operation_3649
Label: Target-Decoy
+   'Target-Decoy' SubClassOf 'Validation of peptide-spectrum matches'
 
Class: http://edamontology.org/operation_3648
Label: Validation of peptide-spectrum matches
+   'Validation of peptide-spectrum matches' SubClassOf 'Peptide database search'
+   'Validation of peptide-spectrum matches' SubClassOf 'Validation'
 
Class: http://edamontology.org/operation_3647
Label: Blind peptide database search
+   'Blind peptide database search' SubClassOf 'Peptide database search'
 
Class: http://edamontology.org/operation_3629
Label: Deisotoping
+   'Deisotoping' SubClassOf 'has input' some 'Mass spectrometry spectra'
+   'Deisotoping' SubClassOf 'Spectral analysis'
 
Class: http://edamontology.org/operation_3631
Label: Peptide identification
+   'Peptide identification' SubClassOf 'has input' some 'Mass spectrometry spectra'
+   'Peptide identification' SubClassOf 'Spectral analysis'
 
Class: http://edamontology.org/operation_3630
Label: Quantification
+   'Quantification' SubClassOf 'has input' some 'Mass spectrometry spectra'
+   'Quantification' SubClassOf 'Spectral analysis'
 
Class: http://edamontology.org/operation_3633
Label: Retention times calculation
+   'Retention times calculation' SubClassOf 'Calculation'
 
Class: http://edamontology.org/operation_3632
Label: Isotopic distributions calculation
+   'Isotopic distributions calculation' SubClassOf 'Calculation'
+   'Isotopic distributions calculation' SubClassOf 'has input' some 'Mass spectrometry spectra'
+   'Isotopic distributions calculation' SubClassOf 'has topic' some 'Proteomics'
 
Class: http://edamontology.org/operation_3635
Label: Labeled quantification
+   'Labeled quantification' SubClassOf 'Quantification'
 
Class: http://edamontology.org/operation_3634
Label: Label-free quantification
+   'Label-free quantification' SubClassOf 'Quantification'
 
Class: http://edamontology.org/operation_3637
Label: Spectral counting
+   'Spectral counting' SubClassOf 'Label-free quantification'
 
Class: http://edamontology.org/operation_3636
Label: MRM/SRM
+   'MRM/SRM' SubClassOf 'Quantification'
 
Class: http://edamontology.org/operation_3639
Label: iTRAQ
+   'iTRAQ' SubClassOf 'Labeled quantification'
 
Class: http://edamontology.org/operation_3638
Label: SILAC
+   'SILAC' SubClassOf 'Labeled quantification'
 
Class: http://edamontology.org/operation_3664
Label: Statistical modelling
+   'Statistical modelling' SubClassOf 'Statistical calculation'
 
Class: http://edamontology.org/operation_3663
Label: Homology-based gene prediction
+   'Homology-based gene prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_3662
Label: Ab-initio gene prediction
+   'Ab-initio gene prediction' SubClassOf 'Gene prediction'
 
Class: http://edamontology.org/operation_3661
Label: SNP annotation
+   'SNP annotation' SubClassOf 'Sequence annotation'
 
Class: http://edamontology.org/operation_3660
Label: Metabolic network modelling
+   'Metabolic network modelling' SubClassOf 'Network simulation'
+   'Metabolic network modelling' SubClassOf 'has topic' some 'Systems biology'
 
Class: http://edamontology.org/operation_3666
Label: Molecular surface comparison
+   'Molecular surface comparison' SubClassOf 'Molecular surface analysis'
+   'Molecular surface comparison' SubClassOf 'Structure comparison'
 
Class: http://edamontology.org/operation_3659
Label: Regression analysis
+   'Regression analysis' SubClassOf 'Statistical calculation'
 
Class: http://edamontology.org/operation_3658
Label: Statistical inference
+   'Statistical inference' SubClassOf 'Statistical calculation'
 
Class: http://edamontology.org/operation_3627
Label: Mass spectra calibration
+   'Mass spectra calibration' SubClassOf 'Spectral analysis'
+   'Mass spectra calibration' SubClassOf 'has input' some 'Mass spectrometry spectra'
 
Class: http://edamontology.org/operation_3628
Label: Chromatographic alignment
+   'Chromatographic alignment' SubClassOf 'Spectral analysis'
+   'Chromatographic alignment' SubClassOf 'has input' some 'Mass spectrometry spectra'
 
Class: http://edamontology.org/operation_3625
Label: Relationship inference
+   'Relationship inference' SubClassOf 'has output' some 'Article data'
+   'Relationship inference' SubClassOf 'has input' some 'Article'
+   'Relationship inference' SubClassOf 'Text mining'
+   'Relationship inference' SubClassOf 'has topic' some 'Literature and reference'
 
Class: http://edamontology.org/operation_3677
Label: Differential binding analysis
+   'Differential binding analysis' SubClassOf 'Nucleic acid sequence analysis'
 
Class: http://edamontology.org/operation_3675
Label: Variant filtering
+   'Variant filtering' SubClassOf 'Sequencing quality control'
+   'Variant filtering' SubClassOf 'Nucleic acid sequence analysis'
 
Class: http://edamontology.org/operation_3672
Label: Gene functional annotation
+   'Gene functional annotation' SubClassOf 'Sequence annotation'