<br/><br/><hr/><br/><h1 style="text-align:center"><a id="Implementations">Existing implementations and annotations with EDAM</a></h1>



<br/><h3><a id="Implementations_EMBOSS">EMBOSS</a></h3>
<p>EMBOSS applications have been annotated using EDAM and these annotations appear in corresponding Web services.
</p>
<p>Annotated WSDL files (and associated XSD data schema) are available from:</p>
<ul><li><a href="http://www.ebi.ac.uk/soaplab/typed/services/list">http://www.ebi.ac.uk/soaplab/typed/services/list</a></li></ul>

<p>You will see a list of service end-points with WSDL URLs. For example:
</p><ul><li><a href="http://www.ebi.ac.uk/soaplab/typed/services/alignment_consensus.cons.sa?wsdl">http://www.ebi.ac.uk/soaplab/typed/services/alignment_consensus.cons.sa?wsdl</a></li></ul>

<p>To see the data schema associated with a WSDL, you must replace <kbd>"?wsdl"</kbd> with <kbd>"?xsd=1"</kbd>, <kbd>"?xsd=2"</kbd> or <kbd>"?xsd=3"</kbd>. For example:
</p><ul><li><a href="http://www.ebi.ac.uk/soaplab/typed/services/alignment_consensus.cons.sa?xsd=1">http://www.ebi.ac.uk/soaplab/typed/services/alignment_consensus.cons.sa?xsd=1</a></li></ul>





<br/><h3><a id="Implementations_BioXSD">BioXSD</a></h3>
<p>The BioXSD XML schema (XSD) defines exchange formats of everyday 
bioinformatics data types. BioXSD aims to serve as the common, canonical
 data model for bioinformatics Web services. It includes commonly used 
types including sequences, sequence annotations, alignments and 
references to resources:</p>
<ul><li><a href="http://bioxsd.org">http://bioxsd.org</a></li></ul><p>BioXSD has been annotated with EDAM concepts.</p>


<br/><h3><a id="Implementations_DRCAT">DRCAT biological resource catalogue</a></h3>
<p>A catalogue of data resources (DRCAT) is being compiled as part of 
the EMBOSS project. Each entry in DRCAT gives metadata on a data 
resource available on the Web. The metadata includes "Query" lines 
describe the type(s) of data available, the data format, data identifier
 (used to query) and a URL from which data can be retrieved. The 
"Query" lines and the resources themselves are annotated with EDAM 
concepts.</p>

<p>A typical entry is shown below:</p>
<p><small>(<i>NB.</i> The format of EDAM ids has not been upgraded yet. Will be done asap.)</small></p>
<pre>ID PDB
Acc DB-0070
Name The RCSB Protein Data Bank
Desc A repository for 3D biological macromolecular structure data.
URL http://www.rcsb.org/pdb/
Cat 3D structure databases
EDAMres EDAM:0000693 | Tertiary structure
EDAMdat EDAM:0000883 | Tertiary structure
EDAMdat EDAM:0002085 | Structure annotation
EDAMfmt EDAM:0001476 | pdb
EDAMfmt EDAM:0001478 | pdbml
EDAMfmt EDAM:0001477 | mmCIF
EDAMfmt EDAM:0002331 | HTML 
EDAMid EDAM:0001127 | PDB ID
Xref SP_explicit | None
Xref SP_FT | None
Xref EMBL_explicit | None
Query EDAM:0002085 | EDAM:0002331 | EDAM:0001127 | http://www.pdb.org/pdb/explore/explore.do?structureId=%s
Query EDAM:0000693 | EDAM:0001476 | EDAM:0001127 | http://www.pdb.org/pdb/files/%s.pdb
Query EDAM:0000693 | EDAM:0001477 | EDAM:0001127 | http://www.pdb.org/pdb/files/%s.cif
Query EDAM:0000693 | EDAM:0001478 | EDAM:0001127 | http://www.pdb.org/pdb/files/%s.xml
Example EDAM:0001127 | 1rbp
Email deposit@deposit.rcsb.org
CCmisc EMBL DR line example "1OSN", /dbxref="PDB:12GS"
Status Referenced
</pre>

<p>DRCAT development will proceed in harmony with bioDBCore, which proposes a community-defined, uniform, generic description of the core attributes of biological databases:</p>

<ul><li><a href="http://biocurator.org/biodbcore.shtml">http://biocurator.org/biodbcore.shtml</a></li></ul>

<p>bioDBCore is under the auspices of the International Society for Biocuration:</p>

<ul><li><a href="http://biocurator.org/home.shtml">http://biocurator.org/home.shtml</a></li></ul>

<p>All enquiries to <a href="mailto:edam-core@elixir-dk.org">EDAM developers</a>.</p>






<br/><h3><a id="Implementations_Bio-jETI">Bio-jETI</a></h3>
<p>Bio-jETI allows 
automatic composition of functional units into software systems 
according to higher-level specifications using EDAM:</p>
<ul><li><a href="http://biojeti.cs.tu-dortmund.de/about.php">http://biojeti.cs.tu-dortmund.de/about.php</a></li></ul>



<br/><h3><a id="Implementations_iHOP">iHOP Web service</a></h3>
<p>The iHOP Web service is annotated with EDAM concepts, either directly or via its use of BioXSD:</p>
<ul><li><a href="http://ws.bioinfo.cnio.es/iHOP/#EMBRACE">http://ws.bioinfo.cnio.es/iHOP/#EMBRACE</a></li></ul>





<br/><h3><a id="Implementations_CBU">CBU Web services</a></h3>
<p>The Web services provided by the Computational Biology Unit (CBU) of the University of Bergen and its affiliated Uni Computing are annotated with EDAM concepts:</p>
<ul><li><a href="http://cbu.bioinfo.no/wsdl">http://cbu.bioinfo.no/wsdl</a></li></ul>





<br/><h3><a id="Implementations_eSysbio">eSysbio</a></h3>
<p>eSysbio was a proof-of-concept prototype workbench for sharing and analysing bioinformatics data using public or private Web services and R scripts. eSysbio used EDAM to annotate and denote the type and format of data items submitted to the system.</p>
<ul><li><a href="http://esysbio.org/about">http://esysbio.org/about</a></li></ul>


<br/><h3><a id="Implementations_SEQwiki">SEQwiki</a></h3>
<p>The SEQanswers wiki is an open catalogue of bioinformatics software tools, non-exclusively focussed on sequencing data analysis. SEQanswers tool wiki uses EDAM for annotation of the listed tools where applicable.</p>
<ul><li><a href="http://seqanswers.com/wiki/Software">http://seqanswers.com/wiki/Software</a></li></ul>



